common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -160,6 +171,36 @@ private boolean handleUrl(final WebView view, final String url) {
                 final PKeyAuthChallenge pKeyAuthChallenge = factory.getPKeyAuthChallengeFromWebViewRedirect(url);
                 final PKeyAuthChallengeHandler pKeyAuthChallengeHandler = new PKeyAuthChallengeHandler(view, getCompletionCallback());
                 pKeyAuthChallengeHandler.processChallenge(pKeyAuthChallenge);
+            } else if (FidoConstants.IS_PASSKEY_SUPPORT_READY && isPasskeyUrl(formattedURL)) {
+                Logger.info(methodTag,"WebView detected request for passkey protocol.");
+                final SpanContext spanContext = getActivity() instanceof AuthorizationActivity ? ((AuthorizationActivity)getActivity()).getSpanContext() : null;
+                final PasskeyFidoChallengeHandler challengeHandler = new PasskeyFidoChallengeHandler(
+                        new CredManFidoManager(view.getContext()),
+                        view,
+                        spanContext,
+                        ViewTreeLifecycleOwner.get(view));
+                try {
+                    final IFidoChallenge challenge = FidoChallengeFactory.createFidoChallengeFromRedirect(url);
+                    challengeHandler.processChallenge(challenge);
+                } catch (final ClientException e) {
```

Comment:
Should we log any error?

Reply:
Changes made here: 2112dd72efe46ae748ab0b2525b21810e46df28a